_data/assetPaths.json

```diff
@@ -1,9 +1,8 @@
 {
-  "admin.js": "/assets/js/admin-44W43542.js",
-  "admin.map": "/assets/js/admin-44W43542.js.map",
+  "admin.js": "/assets/js/admin-77FHK54G.js",
+  "admin.map": "/assets/js/admin-77FHK54G.js.map",
   "app.js": "/assets/js/app-XWR645AF.js",
   "app.map": "/assets/js/app-XWR645AF.js.map",
-  "uswds.js": "/assets/js/uswds-init.js",
-  "styles.css": "/assets/styles/styles-EGZPWJZ2.css",
-  "styles.map": "/assets/styles/styles-EGZPWJZ2.css.map"
+  "styles.css": "/assets/styles/styles-XT7WQ5Z5.css",
+  "styles.map": "/assets/styles/styles-XT7WQ5Z5.css.map"
 }
```

Comment:
I know it wasn't here before, but mind adding an end of file new line here please